x-pack/test/api_integration/apis/security/privileges.ts

```diff
@@ -39,6 +39,7 @@ export default function({ getService }: FtrProviderContext) {
               siem: ['all', 'read'],
               fleet: ['all', 'read'],
               epm: ['all', 'read'],
+              ingestManager: ['all', 'read'],
```

Comment:
@elastic/kibana-security I think this is the only change that triggered codeowners review for security. This PR is going into a feature branch so you may just want to wait until we cut a PR against master to review.